### PR TITLE
Day13 Ex1 NoteSQLite

### PR DESCRIPTION
**Pull request by anhhuy173**
#### Description (Required)
- [x] Create layout 
- [x] Add note model
- [x] Add SQLite
#### Reference (Optional)

### CODE REVIEW

**Comment by at-tamnguyen2** [comment]
@anhhuy173 em check lại chỗ back fragment, từ addnote nếu back thì phải quay về list note, tương tự view note cũng vậy. Còn lại ok rồi. Thanks

**Comment by anhhuy173** [comment]
thanks anh @at-tamnguyen2  dạ để em chỉnh lại.

File: app/src/main/java/vn/asiantech/internship/ui/adapters/NoteAdapter.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            holder.mImgImage.setImageURI(Uri.parse(mNoteList.get(position).getImage()));
+        }
+    }
+
+    @Override
+    public int getItemCount() {
+        return mNoteList.size();
+    }
+
+    class ViewHolder extends RecyclerView.ViewHolder {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/ui/adapters/NoteAdapter.java]: ERROR: Missing a Javadoc comment.

File: app/src/main/java/vn/asiantech/internship/ui/adapters/NoteAdapter.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        private void setData(int position) {
+            Note note = mNoteList.get(position);
+            mTvTime.setText(note.getStringTime());
+            mTvTitle.setText(note.getTitle());
+            mTvContent.setText(note.getContent());
+        }
+    }
+
+    public interface OnItemClickListener {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/ui/adapters/NoteAdapter.java]: ERROR: Missing a Javadoc comment.

File: app/src/main/java/vn/asiantech/internship/ui/note/AddNoteFragment.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import vn.asiantech.internship.R;
+import vn.asiantech.internship.database.NoteDatabase;
+import vn.asiantech.internship.models.Note;
+
+/**
+ * Fragment Add New Note
+ * Created by huypham on 20/06/2017.
+ */
+public class AddNoteFragment extends Fragment {
+    public EditText edtTitle;
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/ui/note/AddNoteFragment.java]: ERROR: Variable 'edtTitle' must be private and have accessor methods.

File: app/src/main/java/vn/asiantech/internship/ui/note/AddNoteFragment.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    public EditText edtContent;
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/ui/note/AddNoteFragment.java]: ERROR: Variable 'edtContent' must be private and have accessor methods.

**Pull request closed without merging**